recipes/polymer/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "polymer" %}
+{% set version = "4.17.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hygeos/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d7ce3e73ba3352501c20268ae2c6e9825d8e9b46b7f72f5ad860ea697a3aff29
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<311 or py>=313]  # Python 3.13 builds require numpy>=2
+  entry_points:
+    - polymer = polymer.cli:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - cython
+    - meson-python
+    - numpy <2  # ensure headers available during cythonization
```

Comment:
conda-forge switched to numpy >= 2 some time ago, is there any reason to constrain numpy to `<= 2` ?